helix-core/src/transaction.rs

```diff
@@ -378,7 +378,9 @@ impl ChangeSet {
             macro_rules! map {
                 ($map: expr, $i: expr) => {
                     loop {
-                        let Some((pos, assoc)) = positions.peek_mut() else { return; };
+                        let Some((pos, assoc)) = positions.peek_mut() else {
+                            return;
```

Comment:
Newer versions of rustfmt can handle let-else statements, so these unrelated blocks of code are modified. Could you reset these changes? I would prefer to update the formatting of these when we bump the MSRV

Reply:
Should be reset now.
Sorry I'm pretty new to open-source.